datasets/monash_tsf/README.md

```diff
@@ -0,0 +1,179 @@
+  - [Discussion of Biases](#discussion-of-biases)
+  - [Other Known Limitations](#other-known-limitations)
+- [Additional Information](#additional-information)
+  - [Dataset Curators](#dataset-curators)
+  - [Licensing Information](#licensing-information)
+  - [Citation Information](#citation-information)
+  - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** [Monash Time Series Forecasting Repository](https://forecastingdata.org/)
+- **Paper:** [Monash Time Series Forecasting Archive](https://openreview.net/pdf?id=wEc1mgAjU-)
+- **Leaderboard:** [Baseline Results](https://forecastingdata.org/#results)
+- **Point of Contact:** [Rakshitha Godahewa](mailto:rakshitha.godahewa@monash.edu)
+
+### Dataset Summary
+
+The first comprehensive time series forecasting repository containing datasets of related time series to facilitate the evaluation of global forecasting models. All datasets are intended to use only for research purpose. Our repository contains 30 datasets including both publicly available time series datasets (in different formats) and datasets curated by us. Many datasets have different versions based on the frequency and the inclusion of missing values, making the total number of dataset variations to 58. Furthermore, it includes both real-world and competition time series datasets covering varied domains.
+
+### Supported Tasks and Leaderboards
+
+- `univariate-time-series-forecasting`: The time series forecasting tasks involves learning the future `target` values of time series in a dataset for the `prediction_length` time steps. The performance of the forecast models can then be validated via the ground truth in the `validation` split and tested via the `test` split.
+
+
+### Languages
+
+## Dataset Structure
+
+### Data Instances
+
+A sample from the training set is provided below:
+
+{
+  'start': datetime.datetime(2012, 1, 1, 0, 0),
+  'target': [14.0, 18.0, 21.0, 20.0, 22.0, 20.0, ...], # <= this target array is a concatenated sample
+  'feat_static_cat': [0], 
+  'feat_dynamic_real': [[0.3, 0.4], [0.1, 0.6], ...], # <= temporal covariates
+  'item_id': '0'
+}
+
+
+### Data Fields
+
+For this univariate regular time series we have:
+
+* `start`: a datetime of the first entry of each time series in the dataset
+* `target`: an array[float32] of the actual target values
+* `feat_static_cat`: an array[uint64] which contains a categorical identifier of each time series in the dataset
+* `feat_dynamic_real`: optional array of covariate features
+* `item_id`: a string identifier of each time series in a dataset for reference
+
+### Data Splits
+
+The datasets are split in time depending on the prediction length specified in the datasets. If no prediction length is specified then we use the following prediction lengths given the frequency of the datasets:
+
+```py
+prediction_length_map = {
+    "S": 60,
+    "T": 60,
+    "H": 48,
+    "D": 30,
+    "W": 8,
+    "M": 12,
+    "Y": 4,
+}
+```
```

Comment:
Let's mention the default number of examples in each dataset (as a table, see [this](https://github.com/huggingface/datasets/tree/master/datasets/imdb#data-splits) for instance), and, similarly to the electricity load diagrams [readme](https://github.com/huggingface/datasets/tree/master/datasets/electricity_load_diagrams#dataset-usage), we should add a `Dataset Usage` section to explain the configuration parameters (`freq`, `prediction_length`, `rolling_evaluations`) that the user can customize.